### PR TITLE
Feature/navigate style

### PR DESCRIPTION
**Pull request by jfrankl**
Andrew, I added the element we discussed. I need to do a bunch of work to make sure it looks right across different devices, but this should be enough of a starting point so you're not blocked.

image

### CODE REVIEW

File: src/nih_wayfinding/app/styles/components/_buttons.scss [hunk 1]
```diff
@@ -4,7 +4,6 @@
   bottom: 0;
   color: #fff !important;
   font-size: $font-size-h4;
-  text-transform: uppercase;
```
Comment (CloudNiner) [inline src/nih_wayfinding/app/styles/components/_buttons.scss:7]: This being a global change was intentional?

Comment (jfrankl) [inline src/nih_wayfinding/app/styles/components/_buttons.scss:7]: Yes, this is intentional.

**Comment by CloudNiner** [comment]
The commit log is weird here. I'd either rebase this branch against develop, and then force push this branch, or create a new branch, cherry pick your commit onto it, and then submit that branch as a new PR. 

Preference to option 1, if you're unsure, chat me and I can help out!

**Comment by jfrankl** [comment]
Not sure what happened there! Just rebased against develop.

**Comment by CloudNiner** [comment]
nice +1

**Pull request merged**